test/generic/Matrix-test.jl

```diff
@@ -133,9 +133,9 @@ end
       M = randmat_with_rank(S, dim, -100:100)
       b = rand(T, -100:100)
 
-      x, d = Generic.solve_fflu(M, b)
+      flag, x, d = Generic.can_solve_with_solution_fflu(M, b)
 
-      @test divexact(M, d)*x == b
+      @test flag && divexact(M, d)*x == b
```

Comment:
```suggestion
      @test divexact(M, d)*x == b
```

Reply:
Why do you want to remove the test for flag?

Reply:
Oh, I see, alright